cli/cortx-prvsnr-cli.spec

```diff
@@ -28,20 +50,24 @@ CORTX Provisioner Command line interface. Provides utilities to deploy CORTX Obj
 
 
 %install
-rm -rf %{buildroot}
+    
+/bin/rm -rf %{buildroot}
+
+/bin/mkdir -p %{buildroot}/opt/seagate/cortx/provisioner/{cli,files/etc,files/.ssh}
 
-mkdir -p %{buildroot}/opt/seagate/cortx/provisioner/{cli,files/etc}
+/bin/cp -pr cli/src %{buildroot}/opt/seagate/cortx/provisioner/cli
+/bin/cp -pr files/etc/yum.repos.d %{buildroot}/opt/seagate/cortx/provisioner/files/etc
 
-cp -pr cli/src %{buildroot}/opt/seagate/cortx/provisioner/cli
-cp -pr files/etc/yum.repos.d %{buildroot}/opt/seagate/cortx/provisioner/files/etc
+/bin/ssh-keygen -o -q -t rsa -b 4096 -a 100 -N '' -f id_rsa_prvsnr
```

Comment:
@sradhanand-pati 
>This is only run when creating a package, not when the end-user installs the package
(from [here](https://rpm-packaging-guide.github.io/#:~:text=A%20SPEC%20file%20can%20be,are%20used%20in%20the%20Body.))

so generated keys would be part of rpm package, I think it's a wrong direction in scope of the secutiry. If you want a dynamic way you need to use scriptlets that are called during actual installation.